.changeset/five-baboons-sparkle.md

```diff
@@ -0,0 +1,20 @@
+---
+'xstate': patch
+---
+
+The `.state` and `._event` values have been added to the meta-object provided as the 3rd argument to service creators:
+
+```ts
+const machine = createMachine({
+  initial: 'someState',
+  states: {
+    someState: {
+      invoke: {
+        src: (_ctx, _e, { state, _event }) => {
```

Comment:
Before deprecating `meta.state`, please consider how we use the state object in some of our guards when we need to assert that the machine is in a particular _parallel_ state:
```javascript
const isModeNormalOrOperationStopped = (context, event, meta) => meta.state.matches('Mode.Normal') || meta.state.matches('Operation.Stopped')
```
AFAIK, this logic cannot be implented using the `in` guards, since they allow for a single target only.